auditbeat/docs/auditbeat-modules-config.asciidoc

```diff
@@ -0,0 +1,38 @@
+[id="configuration-{beatname_lc}"]
+=== Specify which modules to run
+
+To enable specific modules and metricsets, you add entries to the
+`auditbeat.modules` list in the +{beatname_lc}.yml+ config file. Each entry in
+the list begins with a dash (-) and is followed by settings for that module.
+
+The following example shows a configuration that runs the `audit` module with
+the `kernel` and `file` metricsets enabled:
+
+[source,yaml]
+----
+auditbeat.modules:
+
+- module: audit
+  metricsets: [kernel]
+  kernel.audit_rules: |
+    -w /etc/passwd -p wa -k identity
+    -a always,exit -F arch=b32 -S open,creat,truncate,ftruncate,openat,open_by_handle_at -F exit=-EPERM -k access
+
+- module: audit
+  metricsets: [file]
+  file.paths:
+    binaries:
+    - /bin
+    - /usr/bin
+    - /sbin
+    - /usr/sbin
+    conf:
+- /etc
+----
+
+Notice that that the `audit` module is specified twice in the example because
```

Comment:
Actually I'll just remove that sentence.  The reason for the structure is pretty clear just by looking at the example.